cmd/antrea-agent/options.go

```diff
@@ -758,3 +762,52 @@ func (o *Options) validateNodePortLocalConfig() error {
 	}
 	return nil
 }
+
+func validateNodeNetworkPolicyPrivilegedRules(rules []agentconfig.PrivilegedRule) error {
+	if len(rules) == 0 {
+		return fmt.Errorf("no privileged rule is found. The absence of privileged rules may result in network unavailability in the Kubernetes cluster")
+	}
+	regex := regexp.MustCompile(`^\d+(?::\d+)?$`)
+
+	for _, rule := range rules {
+		protocol := strings.ToLower(rule.Protocol)
+		switch protocol {
+		case "tcp":
+			fallthrough
+		case "udp":
+			fallthrough
+		case "sctp":
+			for _, portStr := range rule.Ports {
+				if regex.MatchString(portStr) {
+					if len(strings.Split(portStr, ":")) == 2 {
+						if _, _, err := parsePortRange(portStr, ":"); err != nil {
```

Comment:
Why does it use a different separator? Inconsistency should always be avoided. We don't want to tell users they should use `-` to represent port range in NodePortLocal but `:` in NodeNetworkPolicy.

Reply:
The reason is just that the port range definition like `1:100` can be used directly in iptables. It is also okay to use the separator `-`.